wiki/Modding_Guides/How_to_not_Embarrass_Yourself_in_the_Projects_Subforum/en.md

```diff
@@ -1,12 +1,13 @@
-So osu!forums has this subforum as many know as **[Beatmap Projects](https://osu.ppy.sh/forum/53)**. It might seem intimidating at first but here are the tips and tricks on [How to not Embarrass Yourself in the Projects Subforum](How_to_not_Embarrass_Yourself_in_the_Projects_Subforum "wikilink").
+How to not Embarrass Yourself in the Projects Subforum
+=======================================================
+
+So osu!forums has this subforum as many know as **[Beatmap Projects](https://osu.ppy.sh/forum/53)**. It might seem intimidating at first but here are the tips and tricks on **How to not Embarrass Yourself in the Projects Subforum**.
 
 Do not request songs
 --------------------
 
-[*IMPORTANT: DO NOT POST BEATMAP SONGS REQUESTS HERE* by: LuigiHann](https://osu.ppy.sh/forum/t/23976/)
+[*Important: Do __not__ post Beatmap song requests here* by: LuigiHann](https://osu.ppy.sh/forum/t/23976/)
```

Comment:
ditto (since this is the title of that thread, leave as-is...? imo, this is kinda like a direct quote, so we shouldn't modify much of the spelling/grammar/caps)